apps/dashboard/app/layout.tsx

```diff
@@ -6,34 +6,39 @@ import { Inter_Tight } from "next/font/google";
 import { authOptions } from "./api/auth/[...nextauth]/route";
 import Sidebar from "@/components/side-bar";
 import Header from "@/components/header";
+import { redirect } from "next/navigation";
 
 const inter = Inter_Tight({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+    title: "Create Next App",
```

Comment:
you should work with @bodymindarts to find a way to remove those linting / formatting issue.

it's not efficient to review a PR that is consisting on a lot of formatting changes. it make the diff big and doesn't help to focus on the actual code change